openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -389,7 +389,31 @@
         }
 
         .copy, .icon {
-          color: $white-t;
+          color: $white;
+        }
+      }
+
+      button {
+        opacity: 1;
+      }
+    }
+
+    // STATE: in-progress
+    &.is--in-progress {
+
+      .step__counter:before {
+        color: $uxpl-primary-base;
+      }
+
+      .step__status__value {
+        background: $uxpl-primary-base;
+
+        .icon {
+          display: inline-block;
+        }
+
+        .copy, .icon {
+          color: $white;
```

Comment:
I'd also like this to be a semantic variable name, e.g. ``$copy-inverse-color``.